generators/kubernetes/templates/kubectl-apply.sh.ejs

```diff
@@ -55,3 +55,54 @@ kubectl apply -f monitoring/jhipster-prometheus-cr.yml
 kubectl apply -f monitoring/jhipster-grafana.yml
 kubectl apply -f monitoring/jhipster-grafana-dashboard.yml
 <%_ } _%>
+
```

Comment:
I'm a bit reluctant to have so much complexity in the `kubectl-apply.sh` script. Can you extract all of this code a new `kiali-setup.sh` script.